.travis.yml

```diff
@@ -16,12 +16,14 @@ cache:
 before_install:
   - sudo apt-get install jq
   - wget -O ~/codacy-coverage-reporter-assembly-latest.jar $(curl https://api.github.com/repos/codacy/codacy-coverage-reporter/releases/latest | jq -r .assets[0].browser_download_url)
-  - docker build -t "draft-store:pgsql" ./docker/database/
-  - docker run -d -e DRAFT_STORE_DB_PASSWORD=$DRAFT_STORE_DB_PASSWORD -p $DRAFT_STORE_DB_PORT:5432 draft-store:pgsql
+  - docker build -t "draft-store-database:latest" ./docker/database/
+  - docker run --name draft-store-database -d -e DRAFT_STORE_DB_PASSWORD=$DRAFT_STORE_DB_PASSWORD -p $DRAFT_STORE_DB_PORT:5432 draft-store-database:latest
 script:
+  - ./gradlew installDist
   - ./gradlew build
```

Comment:
is both `installDist` and `build` needed?

Reply:
you're right, fix coming up